SmartDeviceLink/SDLFileManager.m

```diff
@@ -76,10 +84,13 @@ - (instancetype)initWithConnectionManager:(id<SDLConnectionManagerType>)manager
 
     _stateMachine = [[SDLStateMachine alloc] initWithTarget:self initialState:SDLFileManagerStateShutdown states:[self.class sdl_stateTransitionDictionary]];
 
+    _failedFileUploadsCount = [NSMutableDictionary dictionary];
+    _maxFileUploadAttempts = configuration.fileRetryCount ? configuration.fileRetryCount + 1 : 1;
```

Comment:
I think this can just be `_maxFileUploadAttempts = configuration.fileRetryCount + 1;`

Reply:
Fixed